docs/_articles/en/user-guide/metadata-operations.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Metadata Operations
+lang: en
+---
+
+## Source Diff
+The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at diffs for an individual file or directory.
```

Comment:
"diff metadata types" is a bit slangy. I suggest you tighten the whole thing up. Ex:
The Source Diff command makes it easy to visual the changes between files in your local project and the metadata in your org. You can look at diffs...